src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/HtmlController.kt

```diff
@@ -60,23 +65,29 @@ class HtmlController(@Autowired val moveService: MoveService, @Autowired val jou
 
   @RequestMapping("/choose-supplier/serco")
   fun chooseSupplierSerco(model: ModelMap): RedirectView {
+    logger.info("chosen supplier Serco")
+
     model.addAttribute(SUPPLIER_ATTRIBUTE, Supplier.SERCO)
     return RedirectView(DASHBOARD_URL)
   }
 
   @RequestMapping("/choose-supplier/geoamey")
   fun chooseSupplierGeoAmey(model: ModelMap): RedirectView {
+    logger.info("chosen supplier GEOAmey")
+
     model.addAttribute(SUPPLIER_ATTRIBUTE, Supplier.GEOAMEY)
     return RedirectView(DASHBOARD_URL)
   }
 
   @RequestMapping("$MOVES_BY_TYPE_URL/{moveTypeString}")
-  fun movesById(
+  fun movesByType(
```

Comment:
Method name was wrong (though technically it has no impact on behavior as it uses the URL.